app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
```

Comment:
if you use raise operator, you can use try except inside that method to return a string value:
try:
   if...:
        raise SomeExc()
   elif:
         raise SomeOtherExc()
   else:
          return "All good"
except Exception as ex:
      return ex.message #returns the message of the exception